plugins/modules/acos_config.py

```diff
@@ -18,6 +17,8 @@
   - A10 ACOS configurations use a simple block indent file syntax
     for segmenting configuration into sections.  This module provides
     an implementation for working with ACOS configuration sections.
+version_added: '2.9'
+author: Hunter Thompson (@hthompson6)

```

Comment:
Let's strip out the author name. Was originally meant as contact point for merging into core. But we are using collections now so it's not as important.

Reply:
Removed author name from all 3 modules

Reply:
Resolved in PR#7